wled00/json.cpp

```diff
@@ -463,7 +464,7 @@ void serializeState(JsonObject root, bool forPreset, bool includeBri, bool segme
     udpn["send"] = notifyDirect;
     udpn["recv"] = receiveNotifications;
 
-    root[F("lor")] = realtimeOverride;
+    root[F("lor")] = realtimeOverride || (realtimeMode && useMainSegmentOnly);
```

Comment:
This one should stay of course until UI is adapted to not show the live popup in main seg mode.